.github/workflows/ci.yml

```diff
@@ -78,8 +77,3 @@ jobs:
         RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
     - uses: actions-rs/grcov@v0.1
     - uses: codecov/codecov-action@v1
-    - run: cargo fmt -- --check
```

Comment:
I think we should check the format in a different workflow instead of ignoring it. It helps since sometimes ppl forget to format the code before push(I did that several times :grimacing:)

Reply:
You're right, of course. I usually keep it in the lint job, so I somehow expected it to be there. Added it there now.